autoscale_cloudroast/test_repo/autoscale/system/integration/test_system_integration_rcv3.py

```diff
@@ -0,0 +1,757 @@
+"""
+System Integration tests for autoscaling with RackConnect V3 load balancers
+"""
+from test_repo.autoscale.fixtures import AutoscaleFixture
+from cafe.drivers.unittest.decorators import tags
+import random
+import time
+from cloudcafe.common.tools.datagen import rand_name
+import unittest
+import inspect
+import logging
+
+
+log = logging.getLogger("RunnerLog")
+
+
+def line():
+    """Returns the current line number in our program."""
+    # Shamelessly taken from
+    # http://code.activestate.com/recipes/145297-grabbing-the-current-line-number-easily/
+    return inspect.currentframe().f_back.f_lineno
+
+
+class AutoscaleRackConnectFixture(AutoscaleFixture):
+    """
+    System tests to verify lbaas integration with autoscale
+    """
+    @classmethod
+    def tearDownClass(cls):
+        """
+        Class-level teardown.  This releases all the resources acquired by
+        the setUpClass method.
+        """
+        cls.resources.release()
+
+    @classmethod
+    def setUpClass(cls):
+        """
+        Capture the initial state of the shared load balancer pools.
+        Since CloudCafe/unittest invokes this method exactly once
+        for all tests, the time spent by this method amortizes nicely
+        across all tests which can benefit from it.
+        """
+        super(AutoscaleRackConnectFixture, cls).setUpClass()
+
+        cls.cloud_servers_on_node = []
+        cls.private_network = {'uuid': '11111111-1111-1111-1111-111111111111'}
+        cls.public_network = {'uuid': '00000000-0000-0000-0000-000000000000'}
+        cls.rackconnect_network = {'uuid': cls.rcv3_cloud_network}
+        cls.min_servers = 1
+
+        # Since RCV3 pools must be created by human intervention,
+        # verify that at least one exists before proceeding with tests
+        tenant_pools = cls.rcv3_client.list_pools().entity.pools
```

Comment:
Good catch.  I removed `_get_available_pools`.  It wouldn't work in this context anyway because it's a class method instead of an object method.